### ISSUE TITLE
PaddleHub迁移学习在保存模型时会退出程序

### ISSUE THREAD

**Issue by oruchimaru420** [PaddlePaddle/PaddleFormers#1672]
欢迎您反馈PaddleHub使用问题，非常感谢您对PaddleHub的贡献！
在留下您的问题时，辛苦您同步提供如下信息：
- 版本、环境信息
1）PaddleHub和PaddlePaddle版本：请提供您的PaddleHub和PaddlePaddle版本号
PaddleHub==2.1.1 PaddlePaddle-GPU==2.1.3.post110
2）系统环境：请您描述系统类型
Win10家庭中文版 显卡RTX 3080 python==3.7.11
- 复现信息：如为报错，请给出复现环境、复现步骤 
全部代码为：
import paddle
import paddlehub as hub
from paddlehub.vision.segmentation_transforms import Compose, Resize, Normalize
from paddlehub.datasets import OpticDiscSeg
from paddlehub.finetune.trainer import Trainer
transform = Compose([Resize(target_size=(512, 512)), Normalize()])
train_reader = OpticDiscSeg(transform, mode='train')
model = hub.Module(name='ocrnet_hrnetw18_voc', num_classes=2, pretrained=None)
scheduler = paddle.optimizer.lr.PolynomialDecay(learning_rate=0.01, decay_steps=1000, power=0.9,  end_lr=0.0001)
optimizer = paddle.optimizer.Adam(learning_rate=scheduler, parameters=model.parameters())
trainer = Trainer(model, optimizer, checkpoint_dir='./test_ckpt_img_ocr', use_gpu=True)
trainer.train(train_reader, epochs=5, batch_size=5, eval_dataset=None,
              log_interval=10, 
              save_interval=1)
其可以正常开始训练
Bug出现在每次保存训练模型时，python环境就会崩溃，退出到cmd中，且模型保存成功，没有给出任何错误信息。
请帮忙解决！

**Comment by haoyuying** [comment]
您好，请贴出完整的报错信息，方便我们定位问题。

**Comment by oruchimaru420** [comment]
就没有错误信息，只是每次保存完模型后，从python环境中退出，然后必须重新进入python环境，再次运行训练，而且模型保存也是成功的

**Comment by haoyuying** [comment]
您好，我使用您的代码，并未出现该问题。请您辛苦提供复线环境，不然按照您的描述很难定位问题。
![image](https://user-images.githubusercontent.com/35907364/139360904-aefb441a-5923-4c23-9da8-64e5f8435bec.png)

**Comment by oruchimaru420** [comment]
您好，我使用AIStudio运行上述代码也是没问题的。只有在本地WIn10系统下运行才会出现上述问题。
本地环境为：
1）PaddleHub和PaddlePaddle版本：
PaddleHub==2.1.1 PaddlePaddle-GPU==2.1.3.post110
2）系统环境：
Win10 64位 家庭中文版 
显卡RTX 3080 
CUDA 11.0 x64
cudnn 8.04.30 x64
python==3.7.11
Anaconda3==5.3.1 x64 在conda环境下运行
